CocoIDE/cdm8_emu.py

```diff
@@ -20,18 +20,18 @@
 
 
 # Python3 and 2
-from __future__ import absolute_import, division, print_function
+from __future__ import absolute_import, division, print_function, annotations
```

Comment:
So far we target 3.6

Reply:
I was planning to add that since for the parent attribute in one of the files, it would be needed for type hinting.